### PR TITLE
User delete

### PR DESCRIPTION
**Pull request by markkhoo**
Just the 'string' translation corrections

### CODE REVIEW

File: lib/strings.js [hunk 1]
```diff
@@ -408,7 +408,8 @@ const Strings = new LocalizedStrings({
       noGoBack: 'No, regresar',
     },
     logout: {
-      popupText: '¿Estás segura de que quieres cerrar sesión?',
+      popupText:
+        '¿Está usted seguro de que desea eliminar TODOS sus datos guardados?',
```
Comment (francisli) [inline lib/strings.js:412]: @markkhoo I don't understand this change- logging out will not trigger a user account delete, so why are we copying that text here?

Comment (markkhoo) [reply lib/strings.js:412]: I think I replaced the wrong string here. Good catch!

File: lib/strings.js [hunk 2]
```diff
@@ -608,7 +608,7 @@ const Strings = new LocalizedStrings({
       noGoBack: '否，返回',
     },
     logout: {
-      popupText: '您确定要退出吗？',
+      popupText: '您確定要刪除您所有儲存下來的資料嗎？',
```
Comment (francisli) [inline lib/strings.js:611]: @markkhoo if I copy and paste this into Google Translate, it gives me "Are you sure you want to delete all your stored data?" - again, this is in the logout section, so is this the correct change?

**Review by francisli — CHANGES_REQUESTED** [review]
Can you review the last change in here...?

File: lib/strings.js (hunk 2 above)
Comment (markkhoo) [reply lib/strings.js]: you are right!

**Comment by francisli** [comment]
@markkhoo at this point, there are no actual changes to merge in this PR. Can we just close it, then?

**Comment by markkhoo** [comment]
We can just close it yea

**Pull request closed without merging**